### DISCUSSION TITLE
How to block media ?

### DISCUSSION THREAD

**Discussion by zhk0603** [apify/crawlee-python#629] [Q&A]
When using Playwright crawler, how to block media, such as stylesheet, font, image, media

**Comment by janbuchar** [comment]
Hello, and thank you for your interest in Crawlee! Do I understand it right that you want to prevent images and such from loading? Perhaps to save bandwidth and make scraping faster?

**Comment by zhk0603** [comment]
yes

**Comment by janbuchar** [comment]
Once pre-navigation hooks are implemented (#427), I would utilize those to call `page.route` and set up blocking as you like. As of now, I'm not aware of an easy way to do this.

**Comment by B4nan** [comment]
Just one note about this, request interception (`page.route`) means not being able to reuse the resource cache of the browser. You can use this approach to save bandwidth, but it will result in worse overall performance most likely.
